src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,10 +1,52 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+export const NewMovie = (props: any) => {
+  const [count, setCount] = useState(0);
+  const [disabled, setDisabled] = useState(true);
+  const { onAdd } = props;
+  const [movieTitle, setMovieTitle] = useState('');
+  const [movieDescription, setMovieDescription] = useState('');
+  const [movieImageUrl, setMovieImageUrl] = useState('');
+  const [movieImdbUrl, setMovieImdbUrl] = useState('');
+  const [movieImdbId, setMovieImdbId] = useState('');
+
+  const [movie, setMovie] = useState({
+    title: movieTitle,
+    description: movieDescription,
+    imageUrl: movieImageUrl,
+    imdbUrl: movieImdbUrl,
+    imdbId: movieImdbId,
+  });
+
+  useEffect(() => {
+    setMovie(
+      {
+        title: movieTitle,
+        description: movieDescription,
+        imageUrl: movieImageUrl,
+        imdbUrl: movieImdbUrl,
+        imdbId: movieImdbId,
+      },
+
+    );
```

Comment:
Would be better to move it to variable
```suggestion
    const newMovie = {
      title: movieTitle,
      description: movieDescription,
      imageUrl: movieImageUrl,
      imdbUrl: movieImdbUrl,
      imdbId: movieImdbId,
    };
    setMovie(newMovie);
```